config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'users/index'
+  get 'sessions/new'
```

Comment:
Are these routes working and/or necessary? It looks like we have different routes for both of these actions below.